content/en/cosign/installation.md

```diff
@@ -109,6 +109,23 @@ $ crane ls gcr.io/projectsigstore/cosign/ci/cosign
 
 Further details and installation instructions for `crane` available via https://github.com/google/go-containerregistry/tree/main/cmd/crane
 
+## Verifying Cosign Releases
+
+Before using a downloaded Cosign binary, it's important to verify its authenticity to ensure that it hasn't been tampered with. The Cosign binary is signed both with keyless signing and an artifact key. You first need to verify Cosign with the artifact key, since you will need Cosign to verify the keyless signature.
+
+```console
```

Comment:
Can we also include the instructions on how to download tuf-client and initialize the TUF environment? https://github.com/sigstore/fulcio/blob/main/README.md?plain=1#L28-L40

Reply:
> Can we also include the instructions on how to download tuf-client and initialize the TUF environment? https://github.com/sigstore/fulcio/blob/main/README.md?plain=1#L28-L40

okay. would get right into this.